test/system/src/test/java/io/pravega/test/system/MultiControllerTest.java

```diff
@@ -72,6 +74,14 @@ public static void setup() throws MarathonException, ExecutionException {
         final List<String> uris = conUris.stream().filter(uri -> DOCKER_BASED ? uri.getPort() == Utils.DOCKER_CONTROLLER_PORT
```

Comment:
Empty line needed before previous comment line.

Reply:
not using this code anymore.